File: WordGames/notconsecutive/notconsecutive.py
# Program to create a list for all of the letters that do not appear consecutively in a word 

# importing the word list
sow_pods = open("/home/evergreen/UnderDogsAssignments/sowpods")

# Go through each word in the file and add it to the list, words
words = []
double_letters = ["AA", "BB", "CC", "DD", "EE", "FF", "GG", "HH", "II", "JJ", "KK", "LL", "MM",
                  "NN", "OO", "PP", "QQ", "RR", "SS", "TT", "UU", "VV", "WW", "XX", "YY", "ZZ"]

for line in sow_pods:
    words.append(line.strip())

for a in words:
    for b in double_letters:
        if b in a:
            double_letters.remove(b)

print("The letters that do not appear consecutively in words are " + str(double_letters))

File: WordGames/notconsecutive/notconsecutive.js
// Program to discover which letters never appeal consecutively
// Nor Currently working
// Importing the fs module
let fs = require("fs")

// Intitializing the readFileLines with the file
const readFileLines = filename =>
   fs.readFileSync(filename)
   .toString('UTF8')
   .split('\n');

// Calling the readFiles function with file name
let words = readFileLines('/home/evergreen/UnderDogsAssignments/sowpods');

letters = ["AA", "BB", "CC", "DD", "EE", "FF", "GG", "HH", "II", "JJ", "KK", "LL", 
"MM", "NN", "OO", "PP", "QQ", "RR", "SS", "TT", "UU", "VV", "WW", "XX", "YY", "ZZ"]
consecutive_Letters = []

for(let i = 0; i < letters.length; i++) {
    for(let j = 0; j < words.length; j++) {
        if (words[j].includes(letters[i])) {
            letters.splice(i, 1);
            continue
        }}}

console.log("Letters that are not consecutively are " + letters)